pkg/plugin/installer/vcs_installer_test.go

```diff
@@ -18,10 +18,13 @@ package installer // import "k8s.io/helm/pkg/plugin/installer"
 import (
 	"io/ioutil"
 	"os"
+	"path/filepath"
 	"testing"
 
 	"k8s.io/helm/pkg/helm/helmpath"
 
+	"fmt"
```

Comment:
This should go up with the other imports and the extra space should be removed in between external imports